packages/allow-scripts/test/prepare.js

```diff
@@ -0,0 +1,20 @@
+const { execSync } = require('child_process')
+const { readdirSync } = require('node:fs')
+const { join, normalize } = require('node:path')
+
+// Execute allow-scripts command inside each test project directory
+readdirSync(join(__dirname, '../test/projects/'), { withFileTypes: true })
+  .filter((p) => p.isDirectory())
+  .forEach((dir) => {
+    try {
+      execSync(`node ${normalize('../../../src/cli.js')} auto --experimental-bins`, {
+        cwd: join(__dirname, '../test/projects/', dir.name),
+        stdio: 'inherit',
+      })
+    } catch (e) {
+      // ignore expected error
+      if (e.status !== 1) {
+        throw e
+      }
```

Comment:
We should assert something about _why_ it's failing, because it may be failing for an unexpected reason (e.g., bad paths)

Reply:
what's the expected error? there was no error capture in the bash for loop this is replacing.